docs/howto/admin/resource-estimation.rst

```diff
@@ -58,11 +59,13 @@ stop, unlike with RAM.
 
 .. math::
 
-    Server CPU Recommended = (Maximum concurrent users \times Maximum CPU usage per user) + 0.2
+    Recommended\, CPU = (Max\, concurrent\, users \times Max\, CPU\, usage\, per\, user) + 20\%
```

Comment:
I'm a little unsure how the commas will render here (will they render)?

Reply:
They render to leave spaces. The previous code didn't render correctly
Without commas:
<img width="639" alt="Screenshot 2019-04-14 at 21 19 10" src="https://user-images.githubusercontent.com/794502/56098720-02e6e780-5efb-11e9-841b-91dafdbd4845.png">

With commas:
<img width="664" alt="Screenshot 2019-04-14 at 21 17 44" src="https://user-images.githubusercontent.com/794502/56098721-067a6e80-5efb-11e9-9781-f107113f3ad1.png">